susemanager/bin/mgr-setup

```diff
@@ -70,124 +57,31 @@ if [[ -n "$TZ" ]]; then
 fi
 
 TMPDIR="/var/spacewalk/tmp"
-DO_MIGRATION=0
-DO_SETUP=0
-LOGFILE="0"
-WAIT_BETWEEN_STEPS=0
-MANAGER_FORCE_INSTALL=0
 PROGRAM="/usr/lib/susemanager/bin/mgr-setup"
```

Comment:
I think this can be removed as well. It's set later to $0, but used only in removed bits.

Reply:
OK, let's remove it too